common/archiver/filestore/visibilityArchiver_test.go

```diff
@@ -54,7 +55,7 @@ type visibilityArchiverSuite struct {
 	container          *archiver.VisibilityBootstrapContainer
 	testArchivalURI    archiver.URI
 	testQueryDirectory string
-	visibilityRecords  []*visibilityRecord
```

Comment:
This was deprecated by me and the archiver* protos creared so that we can better support json (and other) serialization formats here.